<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playlist_song_add_screen</class>
 <widget class="QWidget" name="playlist_song_add_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="top_w" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="niceColumns_w" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
